resources/views/app/feed.blade.php

```diff
@@ -0,0 +1,15 @@
+@include('app.includes.sections.head')
+
+ <!-- Dropdown Structure -->
+@include('app.includes.sections.top_nav')
+
+<div class="row">
+    @include('app.includes.sections.side_nav')
+    @include('app.includes.feeds.body')
+</div>
+
+@include('app.includes.sections.footer')
+
+
+
```

Comment:
So shouldn't this extend the layout page. instead of having several includes?. We also agreed on renaming it from feed to index.